app/api/business/questions_business.py

```diff
@@ -32,6 +32,18 @@ def get_counts(brief_id, questions=None, answers=None):
     }
 
 
+def is_open_to_all(brief):
```

Comment:
This logic seems a bit different to how `open_to_all` is determined in the `get_brief()` view, which looks like:

```
open_to_all = (
    brief.lot.slug in ['atm', 'specialist'] and
    brief.data.get('openTo', '') == 'all'
)
```

Should it be the same?

Reply:
it should be, and that one is wrong.. because atm should be open to all regardless of openTo

Reply:
atm has both open to "all" and "category" - not all are "all". In some places these two types are treated the same (daily seller email) but not always (determining who can respond).

Reply:
as discussed